### ISSUE TITLE
Make large raffle meter based on divs and not :before and :after

### ISSUE THREAD

**Issue by TJScalzo** [team178/team178.github.io#111]
I think [this](http://codepen.io/whage/pen/ljLHn) might help with this? I'll look into it.

**Comment by TJScalzo** [comment]
I've turned [this](http://codepen.io/whage/pen/ljLHn) into [this](http://codepen.io/TJScalzo/pen/eZBpEQ). Can @SilverSapo, @poojpooj, @anusha2617, and @gisellegk tell me what you think? I feel like it could be implemented fairly easily. We could also (probably) apply a tooltip to each part.

**Comment by gisellegk** [comment]
@TJScalzo Awesome!

**Comment by poojpooj** [comment]
I really like it!!

On Monday, March 14, 2016, Tim Scalzo <notifications@github.com> wrote:

> I've turned this <http://codepen.io/whage/pen/ljLHn> into this
> <http://codepen.io/TJScalzo/pen/eZBpEQ>. Can @SilverSapo
> <https://github.com/SilverSapo>, @poojpooj <https://github.com/poojpooj>,
> @anusha2617 <https://github.com/anusha2617>, and @gisellegk
> <https://github.com/gisellegk> tell me what you think? I feel like it
> could be implemented fairly easily. We could also (probably) apply a
> tooltip to each part.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/team178/team178.github.io/issues/111#issuecomment-196535523>
> .
>

-- 


The documents accompanying this fax or e-mail transmission, including any 
attachments, are for the sole use of the intended recipients and MAY 
contain confidential school, health or other information that is legally 
privileged. The authorized recipient of this information is prohibited from 
disclosing this information to any other party unless required or permitted 
to do so by law or regulation. 

If you are NOT the intended recipient you are hereby notified that ANY 
disclosure, copying, or further distribution of the contents of these 
documents is prohibited. If you have received this information via 
facsimile in error, please notify the sender immediately and arrange for 
the return or destruction of these documents. If information is received 
via e-mail and you are not the intended recipient, please contact the 
sender by e-mail immediately and delete/destroy both the original and the 
reply e-mail message.

**Comment by TJScalzo** [comment]
Okay... I've made an attempt to turn the Pen I made into something that works on the website. I have failed very miserably. I feel like I need someone like @gisellegk who I can talk to while I do it. Otherwise I will make mistakes and forget how things worked when we first made the raffle meter. The other thing I need help with is making the code I wrote work with the small raffle meter.

TL;DR: I'm unable to convert this Pen into code for the website on my own. I need help from @gisellegk if possible.

**Comment by TJScalzo** [comment]
Considering I'm no longer part of the team and the raffle meter was my baby, I'm going to close this issue for now. But if someone has the skill, interest, and time to try and renovate the very base structure of how the raffle meter functions, please feel free to reopen this issue.
